### PR TITLE
Document `influxd-ctl backup -estimate` flag

### LINKED ISSUE #3480: Add note about new info added to the influxd-ctl backup
https://github.com/influxdata/feature-requests/issues/22
https://github.com/influxdata/feature-requests/issues/61

https://docs.influxdata.com/enterprise_influxdb/v1.9/administration/cluster-commands/#copy-shard-status

### PR DESCRIPTION
**Pull request by pierwill**
Closes #3480.

### CODE REVIEW

File: content/enterprise_influxdb/v1.9/tools/influxd-ctl.md [hunk 1]
```diff
@@ -246,6 +246,10 @@ Optional arguments are in brackets.
 
 Name of the single database to back up.
 
+###### [ `-estimate` ]
+
+Provide estimated backup size and progress messages during backup.
```
Comment (davidby-influx) [inline content/enterprise_influxdb/v1.9/tools/influxd-ctl.md:251]: Do we want an example output?

**Review by davidby-influx — APPROVED** [review]
One non-blocking suggestion.

File: content/enterprise_influxdb/v1.9/tools/influxd-ctl.md (hunk 1 above)
Comment (pierwill) [reply content/enterprise_influxdb/v1.9/tools/influxd-ctl.md:251]: If you have one handy, that'd be great!

Comment (davidby-influx) [reply content/enterprise_influxdb/v1.9/tools/influxd-ctl.md:251]: > Backing up node backup_data_0_1:8088, db stress, rp autogen, shard 14
Files: 8 / 9	Bytes: 189543424 / 231921095	Completed: 82% in 22s	Estimated remaining: 3s
Files: 8 / 9	Bytes: 189543424 / 231921095	Completed: 82% in 23s	Estimated remaining: 2s
Files: 9 / 9	Bytes: 231736320 / 231921095	Completed: 100% in 24s	Estimated remaining: 447µs
Done backing up node backup_data_0_1:8088, db stress, rp autogen, shard 14 in 67ms: 42192896 bytes transferred

From:
https://github.com/influxdata/feature-requests/issues/61#issuecomment-932632697

**Pull request merged**